### ISSUE TITLE
test_goodbye_rpc failing

### ISSUE THREAD

**Issue by paulhauner** [sigp/lighthouse#1017]
## Description

I got a failure on `v0.2.0` that I suspect this is a part of the long running "port issues with networking tests" issue:

```
 failures:

---- test_goodbye_rpc stdout ----
thread 'test_goodbye_rpc' panicked at 'should build libp2p instance: Error(Msg("Libp2p was unable to listen on the given listen address."), State { next_error: None, backtrace: InternalBacktrace { backtrace: None } })', beacon_node/eth2-libp2p/tests/common/mod.rs:88:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Test run: https://github.com/sigp/lighthouse/pull/1001/checks?check_run_id=598510019

@pawanjay176 and @AgeManning perhaps you've seen this one too?

**Comment by paulhauner** [comment]
I just had it happen again:

https://github.com/sigp/lighthouse/pull/1022/checks?check_run_id=600217141

**Comment by michaelsproul** [comment]
Same, I had it happen with `test_blocks_by_root_chunked_rpc`: https://github.com/sigp/lighthouse/pull/900/checks?check_run_id=600381121

**Comment by adaszko** [comment]
https://github.com/adaszko/lighthouse/runs/601041923

**Comment by AgeManning** [comment]
To my knowledge this is resolved in v0.2.0

**Comment by AgeManning** [comment]
Was resolved in: https://github.com/sigp/lighthouse/pull/1027
